From b5b2fb03d35bbc49ccf39661e024a5e2136daa00 Mon Sep 17 00:00:00 2001
From: Oleg Rakitskiy <oleg.rakitskiy@gmail.com>
Date: Wed, 13 Nov 2019 21:05:46 +0200
Subject: [PATCH] Fix connection established response

small fix of the issue described here
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=914798

new curl version starting from 7.54 supports suppressing
connect header, so used this options instead of
creating own workaround
---
 url.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/url.ml b/url.ml
index ac185f2..99e73f2 100644
--- a/url.ml
+++ b/url.ml
@@ -56,7 +56,7 @@ let rate_option =
 
 let curl_command options url =
   Printf.sprintf
-    "%s --fail --silent --header \"Pragma: no-cache\" %s %s %s"
+    "%s --fail --silent --header \"Pragma: no-cache\" --suppress-connect-headers  %s %s %s"
     curl_path rate_option (String.concat " " options) (quoted_string url)
 
 let head_command = curl_command ["--head"]
-- 
2.1.4

